cursorless-talon/src/modifiers/position.py

```diff
@@ -1,23 +1,30 @@
+from typing import Any
```

Comment:
Worth a look at this file; a bit cleaner now I think

Reply:
If we are going to remove before and after I definitely think that we should add positions to csv file so user can customize them because start of an end of is to verbose. I also think that we should disable chuck after since if we're going to do this puristicly we're going to do it all the way.